muikku-core-plugins/src/main/resources/META-INF/resources/scripts/gui/muikku-file-field-component.js

```diff
@@ -312,11 +312,11 @@
       this._readonly = readonly;
       if (readonly) {
         $(this._uploader).attr("disabled", "disabled");
-        if (this.element.data('file-count') > 0) {
```

Comment:
Does this work when removing files? As far as i remember removing files does not actually remove element but marks it as removed instead (so it can be restored)